Is node 14 still supported in v5?

Recently while trying to use this package in a meteor project, I received this warning:

npm WARN notsup Unsupported engine for webdav@5.7.1: wanted: {"node":">=16"} (current: {"node":"14.18.3","npm":"6.14.15"})
npm WARN notsup Not compatible with your version of node/npm: webdav@5.7.1

+ webdav@5.7.1
updated 1 package and audited 1415 packages in 17.35s

It did appear to install the packages, but when I try to do this:

import { createClient } from "webdav";

I get a build error:

W20241121-11:37:54.574(-6)? (STDERR) Error: Cannot find module 'webdav'

Is it truly not supported or am I importing it incorrectly?

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "sync"
   ],
   "engines": {
-    "node": ">=16"
+    "node": ">=14"
   },
   "lint-staged": {
     "{source,test}/**/*.{js,ts}": [

